doc/source/io.rst

```diff
@@ -5,25 +5,22 @@
 .. ipython:: python
    :suppress:
 
-   import os
    import csv
-   from pandas.compat import StringIO, BytesIO
-   import pandas as pd
-   ExcelWriter = pd.ExcelWriter
-
+   import os
+   import matplotlib.pyplot as plt
    import numpy as np
-   np.random.seed(123456)
+   import pandas as pd
+   import pandas.io.date_converters as conv
+   from pandas.compat import StringIO, BytesIO
    randn = np.random.randn
    np.set_printoptions(precision=4, suppress=True)
-
-   import matplotlib.pyplot as plt
    plt.close('all')
-
-   import pandas.util.testing as tm
    pd.options.display.max_rows = 15
    clipdf = pd.DataFrame({'A': [1, 2, 3], 'B': [4, 5, 6], 'C': ['p', 'q', 'r']},
                          index=['x', 'y', 'z'])
 
+
+
```

Comment:
are those needed for the validation? I'd say flake8-rst should fail with 3 blank lines